runtime/near-vm/.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.'cfg(all(target_os = "linux", target_env = "gnu"))']
+rustflags = [
+    # Put the VM functions in the dynamic symbol table.
+    "-C", "link-arg=-Wl,-E",
```

Comment:
As far as I am aware this is not necessary for us, we could drop this (this also doesn't get used anyhow, as far as I know.)